### ISSUE TITLE
Infinite loop freezes screen with nasty error message

### ISSUE THREAD

**Issue by asarkar** [scalacenter/scastie#408]
```
Iterator
  .from(1)
  .dropWhile(_ % 2 != 0)
  .toList
```
![infinite](https://user-images.githubusercontent.com/1302775/43369454-80bed658-9323-11e8-80e9-4407a87f229c.jpg)

I understand you want to show an error message, but it can be printed on the console, or if displayed on screen, come with a dismiss (X) button.  A message that won't go away is annoying.

**Comment by MasseGuillaume** [comment]
Good point, it should have an (X) to dismiss it. Currently, you can press (Esc) or Hit the: Clear Annotations button.
